components/review/shared/reviewStepper.tsx

```diff
@@ -94,7 +96,18 @@ export const ReviewStepper: React.FC<Props> = ({ currentStage }) => {
                 <Button
                   className="justify-self-end whitespace-nowrap"
                   size="sm"
-                  onClick={() => setStage?.(getNextStage())}
+                  disabled={
+                    currentStage != ReviewStage.INFO &&
+                    currentStage != ReviewStage.END_SUCCESS &&
+                    (scores == undefined ||
+                      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                      scores.get(currentStage)! <= 0 ||
+                      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Rather than disabling the lint errors, can we add a check that `scores.get(currentStage)` exists to the calculation of `disabled`?